source/extensions/filters/http/ext_proc/http_client/client_base.h

```diff
@@ -22,11 +25,10 @@ class RequestCallbacks {
 class ClientBase {
```

Comment:
Is ClientBase common to both gRPC and HTTP clients?

Reply:
Yes

Reply:
When it should be moved out of the http_client directory, since this implies this is http client specific.

Reply:
done